File: templates/create_formula/languages/typescript/src/formula/formula.ts
import clc from "cli-color"

function Run(input1: string, input2: string, input3: boolean) {
    console.log("Hello World!")
    console.log(clc.green(`You receive ${input1} in text.`));
    console.log(clc.red(`You receive ${input2} in list.`));
    console.log(clc.yellow(`You receive ${input3} in boolean.`));
}

const formula = Run
export default formula

File: templates/create_formula/languages/python3/src/formula/formula.py
#!/usr/bin/python3
from colored import fg, attr


def Run(input1, input2, input3):
    print("Hello World!")
    print(f"{fg(2)}You receive {input1} in text.{attr(0)}")
    print(f"{fg(1)}You receive {input2} in list.{attr(0)}")
    print(f"{fg(3)}You receive {input3} in boolean.{attr(0)}")
